crates/http_client/src/github.rs

```diff
@@ -123,7 +124,10 @@ pub fn build_tarball_url(repo_name_with_owner: &str, tag: &str) -> Result<String
     ))?;
     // We're pushing this here, because tags may contain `/` and other characters
     // that need to be escaped.
+    #[cfg(not(target_os = "windows"))]
     let tarball_filename = format!("{}.tar.gz", tag);
+    #[cfg(target_os = "windows")]
+    let tarball_filename = format!("{}.zip", tag);
```

Comment:
With this change the name of the function is no longer accurate, as `.zip` files are not tarballs.

Reply:
Since my poor English skills, do you have any suggestions for a better name?

Reply:
I see you are handling this, thanks!